### DISCUSSION TITLE
Is it possible to increase the TGP limits?

### DISCUSSION THREAD

**Discussion by AlexRD-42** [seerge/g-helper#3459] [General]
Currently the TGP is limited to 95w for me. I wanted to increase it to 120w so I can remove the 20w dynamic boost. Is this possible? Also, for some reason my power plans have all disappeared and only balanced remains. Can't even create new power plans or duplicate them

**Comment by seerge** [comment]
@illiendev hello, no, that's not possible. All ranges are hardcoded for your device :) I assume you have 2024 model with TDP of 125W ?

**Comment by AlexRD-42** [comment]
I have the GU605MI model, which should be able to run 95w + 20w dynamic boost. Ah, shame really, cause with the dynamic boost transferring power from the CPU to the GPU you can't really calibrate the CPU wattage properly. If you increase it to account for 20w deficit you go over the saturation point for the CPU.

**Reply by seerge** [reply]
@illiendev well, main limitation comes from your VBIOS (that is limited to a certain value), you can check it under Nvidia Control panel.

But I think this is for a good reason, as laptop probably doesn't have enough cooling to deal with higher wattages. And also, since you have 4070 - there is almost no performance gains past 100W mark anyway for it.

**Reply by AlexRD-42** [reply]
I've been testing power limitations to see what runs best, and it's kind of useless due to thermal throttling because the only real difference in setting these power limits is how long it takes to throttle.

Some results completely stump me, because they should yield same results but don't. For example:
Test 1: GPU TDP of 95w (75w + 20w boost) and CPU PL of 80w (resulting in a 60w after boost)
Average of 167 FPS and 5% Low = 147

Test 2: GPU TDP of 95w (90w + 5w boost) and CPU PL of 65w (resulting in a 60w after boost)
Average of 160 FPS and 5% Low = 140

I don't know exactly what to make of it, considering the TGP is the same. But it seems running the GPU below 100w like you said yields better results

**Reply by seerge** [reply]
@illiendev it's absolutely not necessarily GPU is hitting a set power limit. Most likely it runs way below cause it has to share power with CPU or game / app can't utilize more :)

You can monitor actual power consumption using HWinfo

**Reply by AlexRD-42** [reply]
Thanks for the HWInfo tip! I was using other sensor monitoring that didn't give me the whole picture. But now it's making even less sense to me, because the GPU's TDP (i assume it's GPU Power under HWInfo) is way lower than what I set in G-Helper despite the GPU being like 6 degrees below the temperature target. So many variables to watch out for

**Reply by seerge** [reply]
You will rarely see GPU hitting actual 100W+, except this is GPU-only scenario like Furmark or 3dmark or so.
